steward/src/cellars/adaptors/aave_v3.rs

```diff
@@ -250,6 +250,9 @@ fn get_encoded_adaptor_calls(
             MorphoAaveV3DebtTokenV1Calls(params) => {
                 calls.extend(adaptors::morpho::morpho_aave_v3_debt_token_adaptor_v1_calls(params)?)
             }
+            BalancerPoolV1Calls(params) => calls.extend(
```

Comment:
Am I misremembering, or is there a second location that needs these encodings for flash loans? Maybe it's for Aave V2?

Reply:
This is the second location. Aave V3 debt adaptor has flashloan which requires its own copy of the protos/encoding

Reply:
Really it's third* cellar_v2 and *_v2_2 each have one